gui/wxpython/history/tree.py

```diff
@@ -76,16 +76,18 @@ def dayToLabel(self, day):
         if day == OLD_DATE:
             return _("No time info")
 
+        base_date = day.strftime("%B {day_number}").format(day_number=day.day)
```

Comment:
While this manual way of creating date will be always limited, this is not allowing to do things like `1 May`. However, we can allow that even with our simple code. 2b5ee4af21aa783d041d9affd4d75d28a459593f had that right, but it had duplication and format with numbers, to make the order customizable by the translator use (untested):

```suggestion
        month_name = day.strftime("%B")
        base_date = _("{month_name} {day_number}").format(month_name=month_name, day_number=day.day)
```

Reply:
Clear. I tried to reduce the number of lines as much as possible and overlooked this issue with translating. Thanks @wenzeslaus .